docs/source_code_examples.md

```diff
@@ -38,7 +38,7 @@
 
 [https://github.com/johanfredin/psx-dev](https://github.com/johanfredin/psx-dev) -  more Code Examples.
 
-[https://github.com/micklemacklemore/mikesPlaystationStuff](https://github.com/micklemacklemore/mikesPlaystationStuff) -  I'm learning to make Playstation 1 games with C
+[https://github.com/micklemacklemore/playstationAssembly](https://github.com/micklemacklemore/psx_cube_demo) -  I'm learning to make Playstation 1 games with C
```

Comment:
The preview between the square brackets, and the actual URL between the parenthesis aren't the same URL. Is this intentional? Did you mean to have two different entries there instead?

Reply:
My bad, yes I was meant to add two entries there: 

https://github.com/micklemacklemore/playstationAssembly    - An assembly example / tutorial

https://github.com/micklemacklemore/mikesPlaystationStuff     - A C example

I can make another PR request and redo the commit?

Reply:
You can just amend this one PR, that's totally fine :)

Reply:
Done, thanks nicolas!!